orchagent/portsorch.cpp

```diff
@@ -2599,6 +2600,10 @@ bool PortsOrch::createVlanHostIntf(Port& vl, string hostif_name)
     attr.value.chardata[SAI_HOSTIF_NAME_SIZE - 1] = '\0';
     attrs.push_back(attr);
 
+    attr.id = SAI_HOSTIF_ATTR_QUEUE;
+    attr.value.u32 = DEFAULT_HOSTIF_TX_QUEUE;
```

Comment:
Should we have a query to check if SAI supports this and only then set?

Reply:
It's possible that the attribute `SAI_HOSTIF_ATTR_QUEUE` is not supported on all platforms.
So, I think it's a must to query SAI capability.

Reply:
I second to this comment. Not all vendors support his attributes, w/o query will fail on some platforms and cause orchagent exit.